bears/general/LineCountBear.py

```diff
@@ -10,14 +12,25 @@ class LineCountBear(LocalBear):
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Formatting'}
 
-    def run(self, filename, file, max_lines_per_file: int):
+    @staticmethod
+    def get_blank_line_count(file):
```

Comment:
Renaming the function with a leading underscore would be a good practise. See https://www.python.org/dev/peps/pep-0008/ for more details. Also, you can make it an instance method like `_get_blank_line_count(self, file):`